### ISSUE TITLE
Header icon color style goes wrong in API docs

### ISSUE THREAD

**Issue by anqurvanillapy** [react-native-training/react-native-elements#473]
- The docs for `<Header/>` API goes wrong about setting the icon's color
    + `leftComponent={{ icon: 'menu', style: { color: '#fff'} }}`
    + Should be `leftComponent={{ icon: 'menu', color: '#fff' }}`
- I take some kinda reference from `react-native-vector-icons` for I tried a lot changing the icon color but failed at following the docs
- My dependencies fragments:

```json
{
  "dependencies": {
    "react": "^16.0.0-alpha.12",
    "react-native": "^0.46.1",
    "react-native-elements": "^0.13.0",
    "react-native-vector-icons": "^4.1.1"
  }
}
```

- I'm new to this community so I'm willing to receive any corrections or anything I missed here :smile:

**Comment by binoy14** [comment]
@anqurvanillapy I believe your right, the docs don't seem to look correct. I will double check and update the docs. Thanks for reporting!

Happy to help with anything else you have questions on. You can join our slack channel and also ask questions there. https://reactnativetraining.herokuapp.com/

**Comment by Monte9** [comment]
@anqurvanillapy thanks for the issue. This has been fixed in the latest release `v0.14.0`

@caphun make the fixes in this commit https://github.com/react-native-training/react-native-elements/commit/2d1c28eed11be1dce199d90b0fd535e6cbb33190

Thanks for the issue tho. 👏
